python/src/function.cpp

```diff
@@ -147,7 +147,7 @@ void function(py::module& m)
              std::shared_ptr<dolfin::function::Constant>>(
       m, "Constant", "A value constant wrt. integration domain",
       py::buffer_protocol())
```

Comment:
The `py::buffer_protocol` should have been removed earlier. We are not using it.